### ISSUE TITLE
Unsupported state class (no statistics possible)

### ISSUE THREAD

**Issue by bcutter** [home-assistant/android#2847]
**Home Assistant Android version:** 2022.8.0

**Android version:** Several

**Phone model:** Amazon Fire HD tablets

**Home Assistant version:** 2022.6.7

**Last working Home Assistant release (if known):** it never did for those affected entities

**Description of problem:** HA dev tools statistics tab shows issue `The state class of this entity, is not supported.` for several entities from different devices, e. g.
- battery temperature
```
unit_of_measurement: °C
device_class: temperature
```
- battery level
```
unit_of_measurement: %
device_class: battery
```
- storage
```
unit_of_measurement: %
```
- light sensor
```
unit_of_measurement: lx
device_class: illuminance
```

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**

```

```

**Screenshot of problem:**

![grafik](https://user-images.githubusercontent.com/13799156/188289002-d9d294cf-1666-4e4e-8746-4b7ae64af4ca.png)

**Additional information:**

- I see this for a long time (months) and finally decided to report it.
- There was no actual change (HA update, Companion app update etc.) that triggered this issue, it existed since integrating the devices.
- Not all entities are affected, only those of the above listed sensor types.

**Comment by dshokouhi** [comment]
duplicate of https://github.com/home-assistant/core/issues/71065

**Comment by dshokouhi** [comment]
> And it's in the app, not HA Core, right?

HA core for this fix actually

**Comment by dshokouhi** [comment]
> Could you (@dshokouhi e. g.) please
> a) reopen this "closed" issue and

no we will not be reopening an issue that was closed as a duplicate of another issue https://github.com/home-assistant/android/issues/2847#issuecomment-1236229644



> b) tell me when exactly this should have been fixed?

According to this HA core issue its still an issue, still not something the app can solve either. Needs to be solved in HA core.

https://github.com/home-assistant/core/issues/83716

**Comment by bschatzow** [comment]
This issue is off and on for over a year. I just had all of my mobile app sensors show up with Unsupported state class in the statistics tab after the update to 2023.6.

**Comment by bcutter** [comment]
Seems like this is the case after EVERY HA restart, not just after an update. I still see those messages permanently even when devices/HA Companion apps are already connected for a (long) while.

It  just seems we can't get rid of this annoying stuff permanently.

**Comment by bschatzow** [comment]
This was @dshokouhi reply in the discord from my question:

I just looked at some GitHub issues and the  Unsupported state class is still open. Is there additional information I can try and get so the issue can be closed?

"at this point we need a HA core team member to take a look at the issue. We have provided all we can in the issue. The app is providing the correct data to HA core its just losing that data when HA core is restarted."

I have no idea how to get a core team member to look at this issue?
